server/src/test/java/keywhiz/service/permissions/PermissionCheckModuleTest.java

```diff
@@ -21,4 +24,27 @@ class Holder {
     createInjector().injectMembers(holder);
     assertNotNull(holder.permissionCheck);
   }
+
+  @Test
+  public void injectedPermissionCheckAllowsClientSecretCreation() {
+    PermissionCheck permissionCheck = createInjector().getInstance(PermissionCheck.class);
+    assertTrue(permissionCheck.isAllowedForTargetType(newClient(), Action.CREATE, Secret.class));
+  }
```

Comment:
This test failed before the new permission check was added to the permission check list.